qiita_pet/nginx_example.conf

```diff
@@ -19,8 +42,7 @@ http {
 
           # CHANGE ME: This should match the WORKING_DIR in your qiita
           # config. E.g.,
-          # alias /Users/username/qiita/qiita_db/support_files/test_data/working_dir/;
-          alias ;
+          alias /Users/username/qiita/qiita_db/support_files/test_data/working_dir/;
```

Comment:
This line was previously commented out as part of the above comment, while the live setting was 'alias ;'. Just making sure this wasn't a temp change that needs to be changed back.

Reply:
Yup, before we were not actually using this for anything but now we do and we can't have empty alias ...